lib/game.rb

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env ruby
+
+require_relative '../lib/player'
+
+class Game
+  WINNING_SET = [
+    [1, 2, 3],
+    [4, 5, 6],
+    [7, 8, 9],
+    [1, 4, 7],
+    [2, 5, 8],
+    [3, 6, 9],
+    [1, 5, 9],
+    [3, 5, 7]
+  ].freeze
+
+  attr_accessor :player1, :player2, :game_arr
+
+  def initialize
+    @player1 = Player.new('', 'X')
+
+    @player2 = Player.new('', 'O')
+
+    @game_arr = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+  end
+
+  def start_game
+    i = 0
+
+    player_counter = 0
+
+    win = false
+
+    while i < 9 && win == false
+
+      valid_pos = false
+
+      input_req = player_counter.even? ? @player1.name : @player2.name
+
+      ask_input(input_req)
+
+      until valid_pos == true
+
+        user_in = recieve_turns
+
+        puts user_in
```

Comment:
* [ ] No `puts/print` and `get.chomp` outside `main.rb` file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/only_main_print.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))